src/main/java/com/monov/cloud/gateway/data/StudentOrCourseIds.java

```diff
@@ -0,0 +1,11 @@
+package com.monov.cloud.gateway.data;
+
+import lombok.Data;
+
+import java.util.List;
+
+
+@Data
+public class StudentOrCourseIds {
```

Comment:
I would name this ItemIds. In the future you might find a use for this class for another type of entity.

Reply:
done